src/main/resources/liquibase/changeset/06.01.2022_1839_create_authors_table.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="aliaksandr" id="06.01.2022_1839_create_tags_table">
```

Comment:
Primary key should be BIGSERIAL type. Foreign key - BIGINT
Please change this and investigate different

Reply:
Ok